symplyphysics/core/vectors/vectors.py

```diff
@@ -143,5 +144,9 @@ def to_base_vector(self) -> Vector:
 
     @staticmethod
     def from_base_vector(vector: Vector, *,
-        dimension: Optional[Dimension] = None) -> QuantityVector:
-        return QuantityVector(vector.components, vector.coordinate_system, dimension=dimension)
+        dimension: Optional[Dimension] = None,
+        subs: Optional[dict[Expr, Quantity]] = None) -> QuantityVector:
+        components = vector.components
+        if subs is not None:
```

Comment:
Use `if else` one-liner.

Reply:
I would argue that would hinder readability, but on the other hand I'm fine with it